features/step_definitions/steps.rb

```diff
@@ -9,3 +9,15 @@
 Given(/^there is a dish named "([^"]*)" in our database$/) do |dish_name|
   Dish.create(name: dish_name)
 end
+
+Given(/^its dish price is "([^"]*)" in our database$/) do |dish_price|
+  Dish.create(name: dish_price)
```

Comment:
Well, this is not entirely correct...

Reply:
With this the dish will have a `name`of 120 ;-)

Reply:
Haha! Ooops, we did Dish.create(price: dish_price) before, not sure why this is the one we ended up with! x)